test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -164,6 +165,51 @@ public void DnuPack_FrameworkSpecified(string flavor, string os, string architec
 
         [Theory]
         [MemberData(nameof(RuntimeComponents))]
+        public void DnuPack_PdbsGeneratedOnAllPlatforms(string flavor, string os, string architecture)
```

Comment:
Is there a way to test that setting `DNX_BUILD_PORTABLE_PDB` creates a portable pdb when `SupportsPdbGeneration` returns true.